components/core/src/os/process/windows.rs

```diff
@@ -27,6 +28,10 @@ use super::{HabExitStatus, ExitStatusExt};
 
 const STILL_ACTIVE: u32 = 259;
 
+const PROCESS_EXITED: i32 = 0;
```

Comment:
These are private to the module but returned from a public function - they should be public so they can be referenced by name